libs/designer/src/lib/ui/templates/__tests__/templateCard.spec.tsx

```diff
@@ -11,63 +11,68 @@ import React from 'react';
 describe('ui/templates/templatesDesigner', () => {
   let store: AppStore;
   let minimalStoreData: Partial<RootState>;
-  let template1Manifest: Template.Manifest;
-  let template2Manifest: Template.Manifest;
-  let template3Manifest: Template.Manifest;
+  let template1Manifest: Template.TemplateManifest;
+  let template2Manifest: Template.TemplateManifest;
+  let template3Manifest: Template.TemplateManifest;
 
   beforeAll(() => {
     template1Manifest = {
+      id: 'template1Manifest',
       title: 'Template 1',
-      description: 'Template 1 Description',
-      skus: ['standard'],
-      kinds: ['stateful'],
-      images: {},
-      details: {},
+      summary: 'Template 1 Description',
+      skus: ['standard', 'consumption'],
+      workflows: {
+        default: { name: 'default' },
+      },
+      details: {
+        By: '',
+        Type: '',
+        Category: '',
+      },
       artifacts: [
         {
-          type: 'workflow',
-          file: 'workflow.json',
+          type: 'description',
+          file: 'description.md',
```

Comment:
We never have this... this portion would include the maps/schemas right? are you trying to test anything in that aspect? if not then just leave it empty